avocado-setup.py

```diff
@@ -159,6 +159,11 @@ def env_check(disable_kvm):
                 "No dependancy check flag is set, proceeding with bootstrap")
             logger.info("Please install following "
                         "dependancy packages %s", " ".join(not_found))
+        elif args.install_deps:
+            logger.warning("Installing packages %s", " ".join(not_found))
```

Comment:
this can be moved to helper function with logger.debug()

Reply:
this is info one time for all packages, make sense..

Reply:
@sathnaga done, please check